gourmet/__init__.py

```diff
@@ -22,3 +22,12 @@
 # if args.thread_debug:
 #     import threading
 #     thread_debug()
+
+# specify the required GI namespaces version here.
+# 'require_version' only needs to be called once to ensures the namespace gets loaded with the given version
+from gi import require_version
+require_version("Gdk", "3.0")
+require_version('Gst', '1.0')
+require_version("Gtk", "3.0")
+require_version("Pango", "1.0")
+require_version('Poppler', '0.18')
```

Comment:
Poppler might be an exception - it's only used by the PDF export, and only on Linux (free desktop platforms, to be precise). So it's probably nice not to require it to start the application in general.

Reply:
I will move `poppler` and `gst` from `gourmet/__init__.py` as they are not required in general.